doc/Getting_Started.md

```diff
@@ -191,11 +191,11 @@ need one of:
 
 #### `tockloader`
 
-Tock requires `tockloader` version `0.7.1`. To install:
+Tock requires `tockloader` version `0.7.2`. To install:
 
 ```bash
-(Linux): sudo pip3 install tockloader==0.7.1
-(MacOS): pip3 install tockloader==0.7.1
+(Linux): sudo pip3 install tockloader==0.7.2
+(MacOS): pip3 install tockloader==0.7.2
```

Comment:
I think pegging to a specific version invites trouble just like you had. I'd remove the version specifier here. And above rephrase to something like `Tock requires at least version 0.7.2`